_pages/en_US/troubleshooting.txt

```diff
@@ -310,6 +310,8 @@ There is an issue with your `otherapp.bin` file (it is missing, misplaced, or co
 <summary><u>"Could not play"</u></summary>
 
 You have the wrong Soundhax file for your device and region, or your device is incompatible with Soundhax. In the latter case, you should update your device to the latest version and follow [Seedminer](seedminer).
+
+If your device is original 3DS or the data is transfered from original 3DS, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) for assistance.
```

Comment:
what's the rationale behind this one?

Reply:
* add discord link for troubleshooting of "could not play" aside from seedminer for old 3ds, as those console might be unminable, especially that they have such early cart-updated system.
Well, currently, 3ds at launch (original 3ds only ofc) is known to be unminable.
From my test, unless you have very early sound app (v0 specifically), it's not likely to show unplayable, it'll just crash.
(Well, unless they're trying to do soundhax on 11.4.0, sound v4096, which they shouldn't reach soundhax page at first place)
And if you have v0 sound app, it's really likely to be launch 3ds imo. (3ds that have 2.2.0 from factory is likely to have sound v1027, which is not really far from the release date)
From my test again, special build of soundhax should be feasible on those console. It's better for them to try this first before they update and try seedminer (and find that it's unminable and then buy flashcart)

Oh, but after review it again myself, should be original 3DS only, those transfered console should be just soundhax compatible, unless the console itself is also original 3DS and having same issue.